google_health/tests/test_pull_api.py

```diff
@@ -1,129 +1,14 @@
-import pytest
-
-from delphi_utils import read_params
-
 from os.path import join, exists
 from tempfile import TemporaryDirectory
-
-import numpy as np
 import pandas as pd
 from pandas.testing import assert_frame_equal
 
 from delphi_google_health.pull_api import (
-    GoogleHealthTrends,
-    get_counts_states,
-    get_counts_dma,
-    _get_counts_geoid,
     _api_data_to_df,
     _load_cached_file,
     _write_cached_file,
 )
 
```

Comment:
Rather than deleting these tests, I would prefer to:

1. Leave the classes and test names here as-is.
2. Comment out the parts of the test body that cause them to fail.  In the worst case, the non-commented part of the test body can just be `assert True`.
3. File an issue to fix these tests.
4. Leave a TODO comment on each of these tests with the number of the issue you just created.

Since we would like to test the pulling behavior, it would be good for future implementers of this feature to see what the previous intended behavior was.